lib/Serge/Engine/Plugin/parse_yaml.pm

```diff
@@ -173,10 +173,10 @@ sub process_node {
         # skip values starting with __PRESERVE_ANCHOR__ or __PRESERVE_REFERENCE__
         if (($string ne '') && ($string !~ '^__PRESERVE_(ANCHOR|REFERENCE)__')) {
             if ($lang) {
-                my $translated_string = &$callbackref($string, undef, $path, undef, $lang);
+                my $translated_string = &$callbackref($string, undef, undef, undef, $lang, $path);
```

Comment:
If `$path` is used as a unique key, it needs to be exported both as hint (3rd param) and key (6th param).